cclib/method/ddec.py

```diff
@@ -300,6 +327,324 @@ def calculate_refcharges(self):
 
             # In DDEC6, weights of 1/3 and 2/3 are assigned for stockholder and localized charges.
             # (Equation 50 and 58 in doi: 10.1039/c6ra04656h)
-            refcharges[atomi] = (stockholdercharges[atomi] / 3.0) + (localizedcharges[atomi] * 2.0 / 3.0)
-        
+            refcharges[atomi] = (stockholdercharges[atomi] / 3.0) + (
+                localizedcharges[atomi] * 2.0 / 3.0
+            )
+
         return refcharges, localizedcharges, stockholdercharges
+
+    def condition_densities(self):
```

Comment:
Docstring?

Reply:
Looks like I've added the docstring only to the later (dependent) PRs of this. If added here as well. Thanks!